recipes/datalad/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "datalad" %}
+{% set version = "0.9.3" %}
+{% set sha256 = "081a4dba4a046a949744dd94232c88fe22fca4decb4ff77edbac842183824944" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Could this be noarch?

Reply:
rright, this one is actually does not build any extensions itself, so we could make it noarch. BUT it depends on git-annex which is available only for Linux. so it must be built and could be installed on Linux only.  Would specifying `noarch: python` be then "kosher" and not lead to failed build/install attempts due to unavailability of git-annex there?

Reply:
It is kosher. Installs on windows/osx will fail, but they would fail if you put in selectors too. Please see: https://github.com/conda-forge/staged-recipes/issues/4787